modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-updating-cluster-manifests_{context}"]
+= Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster
+
+// TODO: Can we find a better title / more generic than calling out the two resources?
+
+After you have mirrored your image set to the mirror registry, you must install the generated `ImageContentSourcePolicy` and `CatalogSource` resources into the cluster.
+
+The `ImageContentSourcePolicy` resource associates the mirror registry with the source registry and redirects image pull requests from the online registries to the mirror registry. The `CatalogSource` resource is used by Operator Lifecycle Manager (OLM) to retrieve information about the available Operators in the mirror registry.
+
+.Prerequisites
+
+* You have mirrored the image set to the registry mirror in the disconnected environment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the OpenShift CLI as a user with the `cluster-admin` role.
+
+. Apply the generated manifests to the cluster:
```

Comment:
Ok, I see where `results-1639608409` came from, but everywhere else, you call it a results directory, not manifests. It might be easier to see what's going on if you "apply the contents of the results directory" or something similar here or talk about the manifests in the results folder elsewhere.